setup.py

```diff
@@ -13,7 +13,7 @@
 
 
 INSTALL_REQUIRES = [
-    "transformers ~= 4.48.1",
+    "transformers ~= 4.49.0",
```

Comment:
Are we sure we want to do that in this PR?

Reply:
There was a bug in the `WhisperGenerationMixin.detect_language` leading the compilation issue, the PR won't be valid with transformers lower than 4.49.0 unfortunately. Inference-wise, all CIs are green, would it be a problem for the training side?

Reply:
https://github.com/huggingface/transformers/blob/858545047c05a35fde437b2ada3a901844cd1e60/src/transformers/models/whisper/generation_whisper.py#L1574

(use_cache need to be set to False here)